Header header
string state_machine
time launch_time

string IDLE = Idle
string RAIL = Rail
string LAUNCH = Launch
string COAST = Coast
string STOP = Stop